src/lib/io/include/sol-memmap-storage.h

```diff
@@ -269,6 +342,16 @@ sol_memmap_write_bool(const char *name, bool value,
     return r;
 }
 
+/**
+ * @brief Reads an int32_t contents.
```

Comment:
I'm not sure, the advantage of using `int32_t` is that make clear that this is a signed int of 32 bits